docs/AuthTokenProject.md

```diff
@@ -466,6 +467,31 @@ SetVolumes sets Volumes field to given value.
 
 HasVolumes returns a boolean if a field has been set.
 
+### GetTags
+
+`func (o *AuthTokenProject) GetTags() []string`
+
+GetTags returns the Tags field if non-nil, zero value otherwise.
+
+### GetTagsOk
+
+`func (o *AuthTokenProject) GetTagsOk() (*[]string, bool)`
+
+GetTagsOk returns a tuple with the Tags field if it's non-nil, zero value otherwise
+and a boolean to check if the value has been set.
+
+### SetTags
```

Comment:
@ctreatma I think this documentation update was miscommunicated as providing the ability to set and update project tags.  `SetTags` is only giving users the ability to update a Project object.  The POST/PUT/PATCH method specs do not include `tags`.

Reply:
Ah yeah, good catch!  It looks like the API was updated to fully support project tags, including when creating/updating a project, but the spec was only partially updated.  I'll file a ticket to get that fixed.  Technically, it is still possible to send tags on create/update by using `AdditionalProperties`, but it would be preferable to support it as a first-class feature of the generated code.